packages/financial-templates-lib/src/price-feed/BasketSpreadPriceFeed.js

```diff
@@ -112,6 +113,23 @@ class BasketSpreadPriceFeed extends PriceFeedInterface {
     return this._getSpreadFromBasketPrices(experimentalPrices, baselinePrices, denominatorPrice);
   }
 
+  debugHistoricalData(time) {
+    let priceFeedErrorDetails = [];
+    this.allPriceFeeds.map(priceFeed => {
```

Comment:
this is just a nit, but you are using map but also pushing data into the priceFeedErrorDetails array. Consider changing this to forEach for side effects like this, or using reduce (removing need for priceFeedErrorDetails) since you are potentially changing length of array.

Reply:
Good idea, I am not using `map` how you are supposed to :)